libtransmission/announcer.c

```diff
@@ -130,20 +135,43 @@ static int compareStops(void const* va, void const* vb)
         return i;
     }
 
-    /* tertiary key: the tracker's announec url */
+    /* tertiary key: the tracker's announce url */
     return tr_strcmp0(a->url, b->url);
 }
 
 /***
 ****
 ***/
 
+struct tr_scrape_info
+{
+    char* url;
+
+    int multiscrape_max;
+};
+
+static void scrapeInfoFree(void* va)
+{
+    struct tr_scrape_info* a = va;
+
+    tr_free(a->url);
+    tr_free(a);
+}
+
+static int compareScrapeInfo(void const* va, void const* vb)
+{
+    const struct tr_scrape_info* a = va;
+    const struct tr_scrape_info* b = vb;
```

Comment:
Missed the spot (const placement) :)

Reply:
dcd86bb4a3e57a82ad8f22b1386dde9016153f4e